### PR TITLE
Add analytics token

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Add analytics token, true by default to delivery urls

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: Example/Tests/ConfigurationTests/CLDAnalyticsTests.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//  Copyright © 2022 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import Cloudinary
+import XCTest
+class CLDAnalyticsTests: BaseTestCase {
+
+    func test_analyicsString() {
+        let cldAnalytics = CLDAnalytics();
```
Comment (const-cloudinary) [inline Example/Tests/ConfigurationTests/CLDAnalyticsTests.swift:15]: I would add a few more edge cases, invalid values, etc

File: Example/Tests/GenerateUrlTests/UrlTests.swift [hunk 2]
```diff
@@ -89,61 +89,68 @@ class UrlTests: BaseTestCase {
...
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "https://test123-res-2.cloudinary.com/image/upload/test")
     }
 
+    func testAnalyticsTrue() {
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, cdnSubdomain: true, secureCdnSubdomain: true, analytics: true)
+        sut = CLDCloudinary(configuration: config)
+        let url = sut?.createUrl().generate("test")
+        XCTAssertEqual(url, "https://test123-res-2.cloudinary.com/image/upload/test?a_AEAEvAF0")
```
Comment (const-cloudinary) [inline Example/Tests/GenerateUrlTests/UrlTests.swift:151]: Will this test work when swift and SDK versions change?

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift [hunk 3]
```diff
@@ -26,11 +26,11 @@ import Foundation
 import UIKit
 
 internal class CLDNetworkCoordinator: NSObject {
+
+    static let DEFAULT_VERSION =        "3.3.0"
```
Comment (const-cloudinary) [inline Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift:30]: Why not expose `func getVersion()` instead?

File: Example/Tests/ConfigurationTests/CLDAnalyticsTests.swift (hunk 1 above)
Comment (adimiz1) [reply Example/Tests/ConfigurationTests/CLDAnalyticsTests.swift:15]: Will do

File: Example/Tests/GenerateUrlTests/UrlTests.swift (hunk 2 above)
Comment (adimiz1) [reply Example/Tests/GenerateUrlTests/UrlTests.swift:151]: No, should set them explicitly, will fix

File: Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift (hunk 3 above)
Comment (adimiz1) [reply Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift:30]: Checking

Comment (adimiz1) [reply Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift:30]: Internal class, have to expose it somehow to Analytics class to get the current version of the SDK
